spikeinterface_gui/curation_tools.py

```diff
@@ -15,7 +15,7 @@
     "removed_units": []
 }
 
-def adding_group(previous_groups, new_group):
+def adding_group(previous_groups, new_group, unit_dtype=None):
```

Comment:
why this need to handle dtype at several places ?
the controller has already an internal unit_ids.dtype and we could use it when we convertit it back from json no ?

Reply:
this we can remove